### PR TITLE
Create community health files.

### PR DESCRIPTION
**Pull request by IMB11**
### Support Information

This will be linked under all issue and discussion sidebars in the "Support" link.

More information: 

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types

### Discussion Templates

- Player Support Template

image

- Mod Dev Support Template

image

### CODE REVIEW

**Comment by IMB11** [comment]
Ready for review.

File: .github/DISCUSSION_TEMPLATE/mod-dev-support.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+labels:
+  - help wanted
+body:
+  - type: checkboxes
+    id: fabric-api
+    attributes:
+      label: Are you using Fabric API?
+      options:
+        - label: 'Yes'
+        - label: 'No'
```
Comment (modmuss50) [inline .github/DISCUSSION_TEMPLATE/mod-dev-support.yml]: Im not too sure how useful this is, most people will be using it, sometimes without knowing about it.

File: .github/DISCUSSION_TEMPLATE/player-support.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        Wiki](https://fabricmc.net/wiki/player:tutorials:logs_ml:windows)
+
+        - [Finding Your Logs on MacOS - Fabric
+        Wiki](https://fabricmc.net/wiki/player:tutorials:logs_ml:mac)
+        
+        
+
+        It is recommended you upload your log files to https://mclo.gs/ to allow
+        easily readability and to censor any sensitive information such as
+        client tokens or personal information exposed by your operating system.
```
Comment (modmuss50) [inline .github/DISCUSSION_TEMPLATE/player-support.yml]: I have never used this website thus dont really know how much I trust it. I dont think we should be guarenteing that it removes personal infomation.

The logs should never contain personal info, maybe other than your computer's username?

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Modpack Developer Support
+
+Are you a modpack developer having problems trying to create a modpack using Fabric?
+
+<!-- Can be uncommented when the pages are created  -->
+<!-- - [Fabric Wiki - Modpack Guides](https://fabricmc.net/wiki/modpack:start)
+- [Fabric Wiki - Creating A Modpack Using Packwiz](https://fabricmc.net/wiki/modpack:creating_packwiz)
+- [Fabric Wiki - Creating A Modpack Using ATLauncher](https://fabricmc.net/wiki/modpack:creating_atlauncher) -->
+- [Official Fabric Discord - `#player-support` and `#mod-dev` channels](https://discord.gg/v6v4pMv)
```
Comment (modmuss50) [inline SUPPORT.md]: This whole section is a bit pointless atm, can add it back if/when these wiki pages are created.

File: SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+You can also [create a new discussion](https://github.com/orgs/FabricMC/discussions/new?category=mod-dev-support) on the GitHub Organization page.
+
+# Found an issue?
+
+If you have found a bug or issue with a tool in the Fabric toolchain, please report it on the suitable repository.
+
+- [Fabric API](https://github.com/FabricMC/fabric/issues)
+- [Yarn](https://github.com/FabricMC/yarn/issues)
+- [Fabric Loader](https://github.com/FabricMC/fabric-loader/issues)
```
Comment (modmuss50) [inline SUPPORT.md:39]: Maybe add loom here as well 👍

File: SUPPORT.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+A full list of repositories in the Fabric toolchain [can be found here.](https://github.com/orgs/FabricMC/repositories)
```
Comment (modmuss50) [inline SUPPORT.md]: Not too sure how useful this is tbh.

File: .github/DISCUSSION_TEMPLATE/player-support.yml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+
+        - [Fabric Wiki - Player
+        Tutorials](https://fabricmc.net/wiki/player:tutorials:start)
+  - type: input
+    id: logs
+    attributes:
+      label: Minecraft Logs
+      description: >-
+        Upload your logs **(if applicable)** to https://mclo.gs/ and paste the
```
Comment (modmuss50) [inline .github/DISCUSSION_TEMPLATE/player-support.yml]: Maybe dont specify a logs site here?

File: .github/DISCUSSION_TEMPLATE/player-support.yml (hunk 2 above)
Comment (modmuss50) [reply .github/DISCUSSION_TEMPLATE/player-support.yml]: Gist might be an alternative, they are already using github anyway.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

File: .github/DISCUSSION_TEMPLATE/mod-dev-support.yml (hunk 1 above)
Comment (IMB11) [reply .github/DISCUSSION_TEMPLATE/mod-dev-support.yml]: I was thinking the same, i'll probably just chuck it.

**Comment by IMB11** [comment]
image

**Review by modmuss50 — APPROVED** [review]
LGTM, this is something that we can easily change as we see how people use it 👍

**Pull request merged**